### ISSUE TITLE
[FEATURE] Check JS template literal close indent in @html-eslint/indent

### ISSUE THREAD

**Issue by jpradelle** [yeonjuan/html-eslint#462]
Currently html content in template literals is checked properly by rule `@html-eslint/indent`, I would like it to check closure of template literal indent. I think it should be either on same line as opening, or 1 row before HTML root indent.

For example, good:
```js
@customElement('test-indent')
export class TestIndent extends LitElement {
  render() {
    return html`
      ${when(true, () => html`<b>ok</b>`)}
      ${when(true, () => html`
        <b>ok</b>
      `)}
    `;
  }
}
```

And bad
```js
@customElement('test-indent')
export class TestIndent extends LitElement {
  render() {
    return html`
      ${when(true, () => html`
        <b>bad</b>
        `)}

      ${when(true, () => html`
        <b>bad</b>
`)}
    `;
  }
}
```

Not sure about this one
```js
@customElement('test-indent')
export class TestIndent extends LitElement {
  render() {
    return html`
      ${when(true, () => html`
        <b>bad</b>`)}
    `;
  }
}
```

Thanks

**Comment by yeonjuan** [comment]
@jpradelle Thank you for the report! Since we’re checking the indentation at the opening part of the expression, it would make sense to check the closing part as well.

**Comment by yeonjuan** [comment]
Hi @jpradelle It's fixed and published in `0.52.0`

**Comment by jpradelle** [comment]
Thanks, but I think I have a case I disagree with.
Those are good
```javascript
  good() {
    return html`
      ${when(true, () => html`
        <b>ok</b>
      `)}

      ${when(
          true,
          () => html`
            <b>ok</b>
          `
      )}
    `;
  }
```

But for this case an error is reported:
```javascript
  bad() {
    return html`
      ${when(
          true,
          () => html`
            <b>ok</b>
          `)}
      ${when(
          true,
          () => html`
            <b>ok</b>`)}
    `;
  }
```
If I fix it, it does:
```javascript
  bad() {
    return html`
      ${when(
          true,
          () => html`
            <b>ok</b>
      `)}
      ${when(
          true,
          () => html`
      <b>ok</b>`)} <-- another error is reported after on this one asking for previous indent
    `;
  }
```

But I think we want to keep the 3 closing chars
```
`)}
```
at expected indent level of closing back quote, not closing bracket
The same way we can do for functions like this:
```javascript
  foo() {
    this.foo(
        this.bar,
        this.baz(
            this.baaz()));
    this.foo(
        this.bar,
        this.baz(
            this.baaz()
        ));
  }
```
